internal/pkg/cli/cmd/template/test/create.go

```diff
@@ -45,12 +45,22 @@ func CreateCommand(p dependencies.Provider) *cobra.Command {
 			}
 
 			// Options
-			options, err := d.Dialogs().AskCreateTemplateTestOptions(tmpl, d.Options())
+			options, warnings, err := d.Dialogs().AskCreateTemplateTestOptions(tmpl, d.Options())
```

Comment:
Z dialogu se vrátí warnings obdobně jako při použití šablony  kde se vrací informace o nutnosti registrovat oauth.